app/Domain/Cart/Tests/Feature/CartTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace App\Domain\Cart\Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class CartTest extends TestCase
+{
+    use RefreshDatabase;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    protected function tearDown(): void
```

Comment:
the method only call parent's method here

Reply:
Yeah, you are right. Fixed.